src/component/form/Input.jsx

```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types";
+
+import InputRequired from "./InputRequired";
+
+import "./input.scss";
+
+const Input = ({type, placeholder, hasError, onChange}) => {
+
+    return (
+        <div className="form__input__group">
+            <input type={type}
+                   className={`form__control${hasError ? " form__input__error" : ""}`}
+                   placeholder={placeholder}
+                   aria-label={placeholder}
+                   onChange={e => {
+                       onChange(e.target.value);
+                   }}/>
+            {hasError ? <InputRequired/> : null}
+        </div>
+    );
+};
```

Comment:
When there is no JavaScript in the component, no need for curly braces

```suggestion
const Input = ({type, placeholder, hasError, onChange}) => 
     <div className="form__input__group">
          <input type={type}
                 className={`form__control${hasError ? " form__input__error" : ""}`}
                 placeholder={placeholder}
                 aria-label={placeholder}
                 onChange={e => {
                     onChange(e.target.value);
                 }}/>
          {hasError ? <InputRequired/> : null}
      </div>
);
```